cypress/e2e/WebInterface/Measure/QDM EditMeasure/QDMMeasureReferences.cy.ts

```diff
@@ -5,7 +5,7 @@ import { MeasureCQL } from "../../../../Shared/MeasureCQL"
 import { Utilities } from "../../../../Shared/Utilities"
 import { EditMeasurePage } from "../../../../Shared/EditMeasurePage"
 
-let randValue = (Math.floor((Math.random() * 1000) + 1))
+let randValue = Cypress._.random(100)
```

Comment:
This is a simpler way to do random numbers, built into Cypress.
This will be anything from 0 - 100.